shark/iree_utils/cpu_utils.py

```diff
@@ -44,4 +45,12 @@ def get_iree_cpu_args():
         error_message = f"OS Type f{os_name} not supported and triple can't be determined, open issue to dSHARK team please :)"
         raise Exception(error_message)
     print(f"Target triple found:{target_triple}")
-    return [f"--iree-llvmcpu-target-triple={target_triple}"]
+    cpu_count = (
+        get_cpu_count()
+        if shark_args.task_topology_max_group_count is None
+        else shark_args.task_topology_max_group_count
+    )
+    return [
+        f"--iree-llvmcpu-target-triple={target_triple}",
+        f"--task_topology_max_group_count={cpu_count}",
+    ]
```

Comment:
Does this work though? Isn't it a runtime flag and the target is a compile time one